script.js

```diff
@@ -74,8 +105,58 @@ class Library {
       }
     }
   }
+
+  myTime() {
```

Comment:
- [ ] The linter throws a `class-methods-use-this` when you declare class methods like this. But you can fix it by replacing this with
```
static myTime() { 
...
```

Please apply this solution to the other places where the linter produces the same error.